docs/user/communication.rst

```diff
@@ -41,99 +45,196 @@ Exemplary exercise detail page with posts:
 |exercise-posts|
 
 Course-wide Posts & Course Communication Overview
-^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
+-------------------------------------------------
 
-Besides lecture or exercise related questions, Artemis offers a third post type: posts with course-wide topics, such as ``Organization`` or ``Tech Support``.
-These posts can only be created on the course communication overview, which is shown in the screenshot below.
+Besides lecture or exercise related questions, Artemis offers a third post type: posts with course-wide topics, such as
+``Organization`` or ``Tech Support``. These posts can only be created on the course communication overview, which is shown
+in the screenshot below.
 
 The ``Communication`` space of an Artemis course serves as overview for *all* posts in a course.
 Hence, course-wide posts as well as exercise posts and lecture posts are listed.
 Here, users can easily query, sort, and filter existing posts.
 
 |course-posts|
 
+Secluded Communication
+----------------------
+Users of a course can communicate in private via the Messages page. (see image below) The page consists of a collapsible
+Conversation sidebar on the left, where users can search for other participants of the current course and start a conversation
+with them.
+
+If the recipient is browsing another conversation when they receive a new message, an envelope icon appears in their
+Conversation Sidebar, next to the affiliated user who has sent the message. This way, users become aware of the new message
+within that discussion.
+
+The authorities of tutors and instructors are more restricted in the Messages Page compared to the Course Communication
+Overview. Only the authors of messages and replies can edit or delete them.
```

Comment:
This makes it a bit unclear to me if e.g. instructors can see these messages

Reply:
You are right. I have made it clear more clear now.